test/tla/TestGetXWithRows.tla

```diff
@@ -0,0 +1,4 @@
+--------------------- MODULE TestGetXWithRows ---------------------------------
+\* @type: { x: a, b } => b;
+GetX(r) == r.x
```

Comment:
hmmm, shouldn't the type for this be 

```suggestion
\* @type: { x: a, b } => a;
GetX(r) == r.x
```

?

Reply:
Oh yes. It should be as you are saying. I was just testing a related issue and forgot to fix it back: https://github.com/informalsystems/apalache/issues/2102

Reply:
If this was type checking, does it indicate a bug? It's not clear to me how see how `a` and `b` could be unifiable here... Or is it right that a single field of a record can have a row type?